modular_citadel/code/modules/research/designs/medical_designs.dm

```diff
@@ -0,0 +1,58 @@
+///////////////////////////////////////////
+/////////Dedibrillators Disks//////////////
+///////////////////////////////////////////
+
+/datum/design/comp_defib
+	name = "compact defibrillator"
+	desc = "A belt-equipped defibrillator that can be rapidly deployed"
+	id = "comp_defib"
+	build_type = PROTOLATHE
+	build_path = /obj/item/defibrillator/compact
+	materials = list(MAT_METAL=46000, MAT_GLASS = 28000, MAT_GOLD = 26000, MAT_SILVER = 26000, MAT_TITANIUM = 16000)
+	construction_time = 100
+	category = list("Misc")
+	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL
+
+/datum/design/defib_heal
```

Comment:
No.

Reply:
If you want you can make them able to revive people with more damage but they don't get to just magic heal.

Reply:
They already magically heal, in fact it would harm them with this disk, adding in more o2 harm to replace it.